crates/cli/src/main.rs

```diff
@@ -192,6 +192,12 @@ enum Cli {
             help = "Path to .dna.json file [default: dist/<dna-name>.dna.json]"
         )]
         path: Option<PathBuf>,
+        #[structopt(
+            long,
+            short = "x",
+            help = "Property (in the from 'name=value') that gets set/overwritten before calculating hash"
```

Comment:
```suggestion
            help = "Property (in the form 'name=value') that gets set/overwritten before calculating hash"
```

Reply:
You would intuitively think it would be "p". How do you feel about making this a breaking change and making "path" a positional argument so we can use "p" for this?